### ISSUE TITLE
`importBulkRelationships` or `bulkImportRelationships`?

### ISSUE THREAD

**Issue by branks** [authzed/authzed-node#170]
It seems `bulkImportRelationships` is inside of an "experiements" folder whereas `importBulkRelationships` is not which leads me to use `importBulkRelationships`

But, it seems the types are wrong for `importBulkRelationships` as the below code is valid typescript but if I debug it at runtime, it seems `writeStream` is a pending promise

``` 
const writeStream = this.client.importBulkRelationships((err, value) => {
writeStream.write(...)
```

![image](https://github.com/user-attachments/assets/3a7faf93-9ea5-428b-843f-1fb03e78f41a)

**Comment by tstirrat15** [comment]
I see the problem. Let me see about getting a fix.

**Comment by tstirrat15** [comment]
I just released what should be a fix in v1.1.1. Can you give it a try and let me know?
